lib/src/injector_base.dart

```diff
@@ -168,11 +124,12 @@ class Injector {
 
   /// Checks if [T] is actually set.
   void _checkValidation<T>() {
-    var type = T.toString();
+    final type = T.toString();
 
     if (T == dynamic) {
       throw Exception(
-          "No type specified !\nCan not register dependencies for type \"$type\"");
+        "No type specified !\nCan not register dependencies for type \"$type\"",
```

Comment:
let's remove the trailing comma

Reply:
I kinda like this is that case since it doesn't break that weird. More personal pref tbh 🤷